src/common.ts

```diff
@@ -36,7 +36,7 @@ export const TOKEN_QUERY_KEY = 'token' as const;
  */
 export interface RequestParams {
   operationName?: string;
-  query: string;
+  query?: string;
```

Comment:
```suggestion
  query: string;
```

It might be annoying, but the [official GraphQL over HTTP spec requires the `query` parameter](https://graphql.github.io/graphql-over-http/draft/#sel-FALFFBDAAACDJ1pQ).